test/kitchen/test/integration/system-probe-test/rspec/system-probe-test_spec.rb

```diff
@@ -20,6 +20,32 @@
   "pkg/collector/corechecks/ebpf/probe"
 )
 
+TIMEOUTS = {
+  "pkg/network/protocols" => "5m",
+  # disable timeouts for pkg/network/tracer
+  "pkg/network/tracer$" => "0",
+}
+
+DEFAULT_TIMEOUT = "10m"
+
+def get_timeout(package)
+  match_size = 0
+  timeout = DEFAULT_TIMEOUT
+
+  # determine longest match
+  TIMEOUTS.each do |k, v|
+    k = Regexp.new(k)
+    v = String(v)
```

Comment:
why is this necessary? isn't it already a string?

Reply:
just being defensive since values are not strongly typed and someone could add a number or something else in the hash declaration

Reply:
👍